dcos/cluster.py

```diff
@@ -138,7 +138,9 @@ def setup_cluster_config(dcos_url, temp_path, stored_cert):
     cluster_name = cluster_id
     try:
         url = dcos_url.rstrip('/') + '/mesos/state-summary'
-        name_query = http.get(url, toml_config=cluster.get_config())
+        name_query = http.get(url,
```

Comment:
This changes the default timeout for this operation from 5 seconds to 3. Is that desired?